config/helm/patches/deployments/capi-controller-manager.yaml

```diff
@@ -22,3 +22,6 @@ spec:
         - name: metrics
           protocol: TCP
           containerPort: 8080
+        securityContext:
+          seccompProfile:
+            type: RuntimeDefault
```

Comment:
Have you seen this is being deleted a few lines above? Why not just drop the `$patch: delete`? :)